### PR TITLE
Update markup for /desktop/snappy

### LINKED ISSUE #1662: /desktop/snappy/ to VBT

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Update markup for /desktop/snappy to VBT

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/desktop/snappy](http://0.0.0.0:8001/desktop/snappy)
- Check that page layout matches the [pattern guide](https://www.dropbox.com/home/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/5.%20Desktop?preview=desktop-snappy.png)

## Issue / Card

Fixes #1662

### CODE REVIEW

**Comment by yaili** [comment]
Will there be a demo link for design review?

**Comment by barrymcgee** [comment]
@yaili I thought that was now done by the bot - what do I have to do to activate it @WillMoggridge?

**Comment by barrymcgee** [comment]
@yaili Demo: http://www.ubuntu.com-1662-desktop-snappy.demo.haus/desktop/snappy

File: templates/desktop/snappy.html [hunk 1]
```diff
@@ -1,183 +1,183 @@
...
-    <div class="six-col equal-height__item">
+<div class="p-strip--grey is-deep is-bordered">
+  <div class="row">
+    <div class="col-6">
       <h1>A &lsquo;snap&rsquo; is a universal Linux package</h1>
       <p>Snaps work on any distribution or device. Snaps are faster to install, easier to create, safer to run, and they update automatically and transactionally so your app is always fresh and never broken.</p>
 
       <p>The public collection of snaps includes the latest and best apps from GitHub and beyond, so you have the whole world of Linux apps at your fingertips.</p>
-      <p><a class="button--primary" href="http://snapcraft.io/docs/core"><span class="external">Get started with snaps</span></a></p>
+      <p><a class="p-button--brand" href="http://snapcraft.io/docs/core"><span class="external">Get started with snaps</span></a></p>
```
Comment (pmahnke) [inline templates/desktop/snappy.html]: p-link--external

File: templates/desktop/snappy.html [hunk 2]
```diff
@@ -1,183 +1,183 @@
...
-      </div>
+</div>
+
+<div class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>Creating a snap is easy!</h2>
+      <p>Snaps have a very simple internal structure &mdash; you can easily craft them by hand! But the easiest way to build a snap is with snapcraft, which supports building from source and from existing packages. Snapcraft also handles publishing your snaps
+        to the world. Read how to <a class="external" href="http://snapcraft.io/docs/build-snaps/">create a snap</a> and join the snap crafting community &mdash; we have a fun crowd of people who hang out on <a class="external" href="https://rocket.ubuntu.com/channel/snapcraft">Rocket Chat</a>      and on the <a class="external" href="https://forum.snapcraft.io/">snapcraft forum</a>.</p>
```
Comment (pmahnke) [inline templates/desktop/snappy.html]: p-link--external

**Comment by pmahnke** [comment]
@barrymcgee also, the hero logos look too big

**Comment by barrymcgee** [comment]
@pmahnke I have no steer on how big is too big, hopefully @yaili will clarify in design review.

**Comment by yaili** [comment]
Design review:
- [ ] Can the logos in the hero be in a col-5 instead of 6? They also need some spacing between them, 2.5 or 3 rem
- [ ] The code blocks in "How to use snaps from the command line" should be normal code blocks rather than code snippets
- [ ] I take back that the "How to use snaps from the command line" should be in an col-8, can we make it col-9 again instead?
- [ ] And "Creating a snap is easy!" col-7 so it doesn't look like the misalignment is a mistake

**Comment by barrymcgee** [comment]
@yaili Made changes, updated demo: http://www.ubuntu.com-1662-desktop-snappy.demo.haus/desktop/snappy

File: templates/desktop/snappy.html [hunk 3]
```diff
@@ -1,183 +1,185 @@
...
-    <div class="six-col equal-height__item">
+<div class="p-strip--grey is-deep is-bordered">
+  <div class="row">
+    <div class="col-7">
       <h1>A &lsquo;snap&rsquo; is a universal Linux package</h1>
       <p>Snaps work on any distribution or device. Snaps are faster to install, easier to create, safer to run, and they update automatically and transactionally so your app is always fresh and never broken.</p>
 
       <p>The public collection of snaps includes the latest and best apps from GitHub and beyond, so you have the whole world of Linux apps at your fingertips.</p>
-      <p><a class="button--primary" href="http://snapcraft.io/docs/core"><span class="external">Get started with snaps</span></a></p>
+      <p><a class="p-button--brand p-link--external" href="http://snapcraft.io/docs/core"><span class="external">Get started with snaps</span></a></p>
```
Comment (pmahnke) [inline templates/desktop/snappy.html]: this should be

```html
<p><a class="p-button--brand" href="http://snapcraft.io/docs/core"><span class="p-link--external">Get started with snaps</span></a></p>
```
image

**Comment by barrymcgee** [comment]
@pmahnke Made that final change, can you approve pls?

**Pull request merged**